src/demo/containers/playground/NewsFeedDemo.tsx

```diff
@@ -1,18 +1,34 @@
+import TableFeedCards from 'lib/containers/TableFeedCards'
 import * as React from 'react'
+import { Container } from 'react-bootstrap'
 import RssFeedCards from '../../../lib/containers/RssFeedCards'
 
-export const NewsFeedDemo:React.SFC = () => {
+export type NewsFeedDemoProps = {
+  token?: string
+}
+
+export const NewsFeedDemo: React.FunctionComponent<NewsFeedDemoProps> = ({
```

Comment:
News Feed Demo now takes the session token (optionally), and shows both the RSS and Table-based components.